### PR TITLE
Add documentation for bypassing the SchemaForm

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
This PR adds the documentation for https://github.com/department-of-veterans-affairs/vets-website/pull/18096 and https://github.com/department-of-veterans-affairs/vets-website/pull/18196.

## Testing done
Tested that it builds the page and adds it to the sidebar. See the screenshot below.

## Screenshots
image

### CODE REVIEW

**Review by theodur — APPROVED** [review]

File: packages/documentation/src/pages/forms/bypassing-schemaform.mdx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  chapters: {
+    chapterOneName: {
+      // Chapter config here...
+      pages: {
+        mySchemalessPage: {
+          path: 'my-schemaless-page',
+          title: 'Bypassing the SchemaForm',
+          CustomPage,
+          CustomPageReview,
+          schema: { // This does still need to be here or it'll throw an error
```
Comment (mpelzsherman) [inline packages/documentation/src/pages/forms/bypassing-schemaform.mdx:35]: Is there any way to make `schema` optional?

**Review by adamwhitlock1 — APPROVED** [review]
Looks pretty understandable to me! Great job

File: packages/documentation/src/pages/forms/bypassing-schemaform.mdx (hunk 1 above)
Comment (cvalarida) [reply packages/documentation/src/pages/forms/bypassing-schemaform.mdx:35]: There probably is, but I didn't dig into what all we'd need to change to make this happen. When I tried omitting it, it threw some errors, so I tried leaving it functionally empty and that did the trick.

My hunch is that changing the library to make `schema` and `uiSchema` truly optional would be kind of a lot of work, but I don't know that for sure, to be fair.

**Comment by bkjohnson** [comment]
I'm merging this for Chris since he's on vacation.

**Pull request merged**